lib/CustomFields/utils/useCustomFieldsFetch.js

```diff
@@ -29,9 +29,7 @@ const useCustomFieldsFetch = (okapi, backendModuleId, entityType) => {
       }
     };
 
-    if (backendModuleId) {
```

Comment:
This check shouldn't be removed. It's possible that the backend doesn't manage to send the response to the `/_/proxy/tenants/diku/modules` request before `useCustomFieldsFetch` is called. This mainly applies to situations when a user visits a page, that fetches custom fields, using a direct link (`/settings/users/custom-fields`, `/users/preview/some-id`, etc). In such cases, this change will lead to making a request for custom fields even though we don't know the `backendModuleId` yet.

Reply:
Reverted this change in a new commit